.chronus/changes/vscode-restart-lsp-2024-9-30-20-29-32.md

```diff
@@ -0,0 +1,8 @@
+---
+changeKind: fix
+packages:
+  - typespec-vscode
+---
+
+1. TypeSpec Language Server would be restarted with new settings when setting "typespec.tsp-server.path" is changed
```

Comment:
create 2 different changelog file for this, first one is a feature not a fix as well

Reply:
done